{"nick":"rorro","reason":"Ping timeout: 256 seconds","date":"2018-01-06T00:33:43.547Z","type":"quit"}
{"nick":"Arcarc","reason":"Ping timeout: 248 seconds","date":"2018-01-06T00:34:39.863Z","type":"quit"}
{"nick":"Tennis","date":"2018-01-06T00:53:51.796Z","type":"join"}
{"nick":"naiveai","date":"2018-01-06T01:08:29.734Z","type":"join"}
{"nick":"polonel","date":"2018-01-06T01:09:10.874Z","type":"join"}
{"nick":"polonel","reason":"Client Quit","date":"2018-01-06T01:11:40.911Z","type":"quit"}
{"nick":"polonel","date":"2018-01-06T01:12:01.825Z","type":"join"}
{"nick":"polonel","reason":"Client Quit","date":"2018-01-06T01:13:29.133Z","type":"quit"}
{"nick":"dman777_alter","date":"2018-01-06T01:49:28.297Z","type":"join"}
{"nick":"dman777_alter","message":"following express.js documentation, I made my last route to handle errors. This works great. Except when I do a DELETE http action. Since it involves 2 reqeusts(fetch/DELETE), I think it is hitting the error route when it shouldn't(because I am successfully sending a res.status(200) in the promisee)","date":"2018-01-06T01:49:34.847Z","type":"message"}
{"nick":"dman777_alter","message":"dpaste.co17CJN55m/","date":"2018-01-06T01:49:39.862Z","type":"message"}
{"nick":"dman777_alter","message":"am I correct in this behavior? otherwise, I can't figure it out since there are no errors","date":"2018-01-06T01:49:44.059Z","type":"message"}
{"nick":"tellnes","reason":"*.net *.split","date":"2018-01-06T02:26:49.026Z","type":"quit"}
{"nick":"tellnes_","date":"2018-01-06T02:28:33.579Z","type":"join"}
{"nick":"Tennis","reason":"Read error: Connection reset by peer","date":"2018-01-06T03:06:41.879Z","type":"quit"}
{"nick":"ddelrio1986","date":"2018-01-06T03:22:42.994Z","type":"join"}
{"nick":"ddelrio1986","reason":"Client Quit","date":"2018-01-06T03:23:06.651Z","type":"quit"}
{"nick":"dman777_alter","reason":"Quit: Lost terminal","date":"2018-01-06T03:36:40.461Z","type":"quit"}
{"nick":"Soopaman","date":"2018-01-06T03:41:32.056Z","type":"join"}
{"nick":"Soopaman","reason":"Max SendQ exceeded","date":"2018-01-06T03:41:36.967Z","type":"quit"}
{"nick":"Soopaman","date":"2018-01-06T03:41:56.985Z","type":"join"}
{"nick":"cwms","date":"2018-01-06T04:21:02.045Z","type":"join"}
{"nick":"cwms","reason":"Quit: Page closed","date":"2018-01-06T04:35:23.155Z","type":"quit"}
{"nick":"Roughy","reason":"Quit: Meadow Fresh milk","date":"2018-01-06T04:40:50.129Z","type":"quit"}
{"nick":"lpin","date":"2018-01-06T06:13:42.458Z","type":"join"}
